### ISSUE TITLE
(FRESH INSTALL) All of a sudden I get "Play Error" for anything I try to play

### ISSUE THREAD

**Issue by GOvEy1nw** [croneter/PlexKodiConnect#1203]
## Help yourself
* I did try to restart Kodi :-)
* I checked the [PKC Frequently Asked Questions on the PKC wiki](https://github.com/croneter/PlexKodiConnect/wiki/faq)
* I did try to reset the Kodi database by going to `PKC Settings -> Advanced -> "Reset the database and optionally reset PlexKodiConnect"` and then hitting YES, NO
* I did check the [existing issues on Github](https://github.com/croneter/PlexKodiConnect/issues)

**YES I DID ALL THE ABOVE**

## Describe the bug
Based on my previous issue report, I followed your instructions & did a fresh install of Kodi. The only think I did was obviously install the PKC repository & PKC itself & get it set up. It scans & adds all the movies & tv shows fine, but then whenever I try to play something I am getting "play error" via PKC, everything works fine in normal Plex on the same device though.

## To Reproduce
N/A since all I'm doing is trying to play a movie or show like normal.

## Expected behavior
The Movie or TV show to just play when click as before.

## You need to attach a KODI LOG FILE!
https://pastebin.com/sHjyYyCB

    
I am aware that I can delete Plex tokens that I accidentially posted by following the [instructions on the PKC wiki](https://github.com/croneter/PlexKodiConnect/wiki/How-to-Report-A-Bug#i-published-my-plex-token-to-some-forum-or-github-anyone-can-now-access-my-plex-server)

**Comment by croneter** [comment]
Hmmm, something is definitely off on the Plex Media Server side of things:
```
WARNING: PLEX.download: Unknown answer from PMS https://192-168-1-4.d505e3d0e637442eac27e8175a6a2da2.plex.direct:32400/playQueues?shuffle=0&repeat=0&type=movie&uri=server%3A%2F%2F7cbeca6073d6e079d11fdfc4bfd254fda3e084de%2Fcom.plexapp.plugins.library%2Flibrary%2Fmetadata%2F647&includeChapters=1 with status code 500: <html><head><title>Internal Server Error</title></head><body><h1>500 Internal Server Error</h1></body></html>
```
Could you please also share the PMS log file?

**Comment by GOvEy1nw** [comment]
Strange! Ok I've just rebooted the NAS I run Plex on and then tried to play something through PKC so hopefully it's a small enough log for you to easily find the issue. here's the log file: https://pastebin.com/pU3HqCU7

**Comment by croneter** [comment]
```
Jul 19, 2020 10:52:03.441 [0x7f03b6b7d700] ERROR - Soci Exception handled: sqlite3_statement_backend::loadOne: database disk image is malformed
```
Ouch. 

Your Plex database is malformed. Please take this up with the Plex team...

**Comment by croneter** [comment]
Any news here?

**Comment by GOvEy1nw** [comment]
Apologies I didn't respond here, to be honest, I've actually moved over to Emby to test that as I've had the Plex database corrupt on me a few times in the past and while I'm a lifetime Plex pass owner I'm not a fan of the direction they're going and how their own apps still have bugs/issues that haven't been fixed for years etc.

But for those reading, is like to confirm that this issue wasn't PKC's fault and was due to the database corruption of Plex itself. The PKC add-on is actually what stopped me from moving from Plex sooner as it's a fantastic add-on to integrate Plex into the Kodi Library. While using it I never had any issues and it played everything via direct play as the upside to using Plex in Kodi is the fantastic and robust Kodi player.

So yea, I may have moved to Emby (though I have decided to use Plex as my 'learning library' so they're separated a bit more), so far Emby is working great, but that doesn't mean I won't be back as it's still early days. Moving to Emby was a personal choice, but the PKC add-on is the bomb and makes the Plex experience MUCH MUCH better than the default Plex experience!

Thank you @croneter for your hard work on this awesome add-on!
